### PR TITLE
Update static site docs

### PR DESCRIPTION
**Pull request by pikinier20**

### CODE REVIEW

File: _overviews/scala3-scaladoc/static-site.md [hunk 1]
```diff
@@ -174,30 +178,43 @@ If the title is specified multiple times, the priority is as follows (from highe
...
+It results in static site containing documents as well as blog. It also contains custom layouts and assets. The structure of rendered documentation can be based on file system but it can also be overriden by YAML configuration.
 
-You can refer to the assets from within any of the files using markdown links:
+### Mapping directory structure
 
-```
-This is my blog post. Here is the image ![](my_image.png) and here is my [pdf](my_file.pdf)```
+Using YAML configuration file, we can define how the source directory structure should be transformed in target directory structure.
+
+Take a look at following subsection definition:
```
Comment (julienrf) [inline _overviews/scala3-scaladoc/static-site.md:210]:
```suggestion
Take a look at the following subsection definition:
```

File: _overviews/scala3-scaladoc/static-site.md [hunk 2]
```diff
@@ -174,30 +178,43 @@ If the title is specified multiple times, the priority is as follows (from highe
...
+```yaml
+- title: Some other subsection
+  index: abc/index.html
+  directory: custom-directory
+  subsection:
+    - page: abc2/page1.md
+    - page: foo/page2.md
+```
+This subsection shows the ability of YAML configuration to map the directory structure.
+Even though the index page and all defined children are in different directories, they will be rendered in `custom-directory`.
```
Comment (julienrf) [inline _overviews/scala3-scaladoc/static-site.md:220]:
```suggestion
Even though the index page and all defined children are in different directories, they will be rendered in `custom-directory`. The source page `abc/index.html` will generate a page `custom-directory/index.html`, the source page `abc2/page1.md` will generate a page `custom-directory/page1.html`, and the source page `foo/page2.md` will generate a page `custom-directory/page2.html`.
```

**Comment by julienrf** [comment]
Is this ready for review?

**Comment by pikinier20** [comment]
> Is this ready for review?

Now it's ready :)

**Comment by julienrf** [comment]
I see several comments/suggestions not taken into account. Could you please address them?

**Pull request merged**